iroha_data_model/src/query.rs

```diff
@@ -78,27 +77,28 @@ pub struct QueryResult(pub Value);
 impl QueryRequest {
     /// Constructs a new request with the `query`.
     pub fn new(query: QueryBox) -> Self {
+        let timestamp_ms = SystemTime::now()
+            .duration_since(SystemTime::UNIX_EPOCH)
+            .expect("Failed to get System Time.")
+            .as_millis();
+        let timestamp_ms = Compact(timestamp_ms);
         QueryRequest {
-            timestamp: SystemTime::now()
-                .duration_since(SystemTime::UNIX_EPOCH)
-                .expect("Failed to get System Time.")
-                .as_millis()
-                .to_string(),
+            timestamp_ms,
             query,
         }
     }
 
     /// `Hash` of this request.
     pub fn hash(&self) -> Hash {
         let mut payload: Vec<u8> = self.query.clone().into();
-        payload.extend_from_slice(self.timestamp.as_bytes());
+        payload.extend_from_slice(&self.timestamp_ms.0.to_le_bytes());
```

Comment:
We have a rule in our style guide not to access inner elements by position index, please destructure it instead.